lisa/features/network_interface.py

```diff
@@ -28,6 +28,7 @@ def switch_sriov(
         raise NotImplementedError
 
     def is_enabled_sriov(self) -> bool:
+        return True # FIXME
```

Comment:
Why is it needed? The base class shouldn't be instanced, so this method can be implemented in subclasses.